bin/p2-rm/main.go

```diff
@@ -80,7 +80,11 @@ func handlePodRemoval(consulClient consulutil.ConsulClient, labeler labels.Appli
 		}
 	}
 
-	fmt.Printf("%s: successfully removed %s\n", rm.NodeName, rm.PodID)
+	if rm.NodeName != "" {
```

Comment:
at first, I was going to leave a comment of the form "But the condition at the top says `if *podUniqueKey != ""`, and now it is not clear how the two are related", but then I realized that even if that were changed, the condition here would be something like `if rm.PodUniqueKey != ""`, which isn't necessarily better. So I guess I have no change to suggest